modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/odata/filter/expression/PredicateExpressionVisitorImpl.java

```diff
@@ -207,13 +209,37 @@ private Predicate _contains(Object fieldName, Object fieldValue) {
 						objectField.getBusinessType() + " field");
 			}
 
+			Table<?> table = _objectFieldLocalService.getTable(
+				_objectDefinitionId, objectField.getName());
+
 			return table.getColumn(objectField.getDBColumnName());
 		}
 		catch (PortalException portalException) {
 			return ReflectionUtil.throwException(portalException);
 		}
 	}
 
+	private ObjectField _getObjectField(String objectFieldName)
+		throws NoSuchObjectFieldException {
+
+		for (ObjectField objectField :
+				_objectFieldLocalService.getObjectFields(_objectDefinitionId)) {
+
+			if (StringUtil.equals(
+					objectField.getBusinessType(),
+					ObjectFieldConstants.BUSINESS_TYPE_RELATIONSHIP) &&
+				StringUtil.endsWith(
+					objectField.getName(),
+					StringPool.UNDERLINE + objectFieldName)) {
+
+				return objectField;
+			}
+		}
+
+		throw new NoSuchObjectFieldException(
+			"No ObjectField exists with the name " + objectFieldName);
```

Comment:
Can we turn "ObjectField" into "object field" instead?

Reply:
Sure!

Reply:
Done!